lib/ansible/modules/cloud/azure/azure_rm_postgresqldatabase.py

```diff
@@ -199,6 +200,7 @@ def exec_module(self, **kwargs):
                 if not self.check_mode:
                     self.delete_postgresqldatabase()
             else:
+                self.fail("Database properties cannot be updated without setting 'force_update' option")
                 self.to_do = Actions.NoAction
```

Comment:
Note: this is dead code because ```self.fail()``` will call ```sys.exit()```.  You should probably either remove it or move it to happen before the ```self.fail()``` call.